nodebuilder/header/config.go

```diff
@@ -74,3 +59,8 @@ func (cfg *Config) trustedHash(net params.Network) (tmbytes.HexBytes, error) {
 	}
 	return hex.DecodeString(cfg.TrustedHash)
 }
+
+// ValidateBasic performs basic validation of the config.
+func (cfg *Config) ValidateBasic() error {
+	return nil
+}
```

Comment:
True! I would like to remove this actually then

Reply:
Either we:
Standardize ValidateError and put it everywhere in baseComponents as an invoke
OR
Only have ValidateError where it is already being used (I actually prefer this)

What do you guys both think? @renaynay @Wondertan

Reply:
I think we should leave it standardised as we have it.

Reply:
Similarly, we should standardise [this](https://github.com/celestiaorg/celestia-node/pull/997#discussion_r971265668) though

Reply:
still disagree with that

Reply:
Pls elaborate

Reply:
It is a subjective opinion. It feels icky to do this until the moment its needed. I feel strongly about this type of thing.

Reply:
I feel strongly about consistency, and if do keep empty Validate - we do pas NodeTypes to the DefaultConfig func

Reply:
We have conflicting opinions and probably are both not open to changing them here because its more a principles thing than anything objective. How do we move forward? Rene and I both agree it should stay as it is, so when can you override? Maybe this kind of thing should be documented somewhere

Reply:
Consistency/Standardization is a diffferent thing than YAGNI. If it is being used nowhere, it doesnt make sense to have it.

Reply:
@Wondertan I agree with ryan here -- let's please not hold up the PR with this.